corehq/apps/translations/integrations/transifex/client.py

```diff
@@ -5,23 +5,18 @@
 import polib
 import requests
 from memoized import memoized
+from transifex.api import TransifexApi
 
-from corehq.apps.translations.integrations.transifex.const import (
-    API_USER,
```

Comment:
nit: this constant isn't needed anymore and can be removed?

Reply:
Yes, Transifex API v3 authentication uses only an API key. https://developers.transifex.com/reference/api-authentication